word_search.py

```diff
@@ -0,0 +1,102 @@
+'''
+Created on 9 June 2017
+
+@author: Eva
+'''
+from queue import Queue
+
+class WordSearch:
+    
+    def __init__(self):
+        return
+        
+    def words_found_in_grid(self, rowNo, columnNo, grid, dictionary):
+        """Uses bfs to find valid words from a grid according to a given dictionary 
+        
+            Args:
+                rowNo: number of rows in the grid
+                columnNo: number of columns in the grid
+                grid: two-dimensional array of characters
```

Comment:
If you get the two dimensional array, you do not actually need to get also `rowNo` and `columnNo` because you can get that from `grid` using `len()`.